adapters/handlers/rest/handlers_index_stats.go

```diff
@@ -0,0 +1,89 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package rest
+
+import (
+	"encoding/json"
+	"net/http"
+	"strings"
+
+	"github.com/weaviate/weaviate/adapters/handlers/rest/state"
+	"github.com/weaviate/weaviate/adapters/repos/db"
+	"github.com/weaviate/weaviate/entities/schema"
+)
+
+func setupIndexStatsHandlers(appState *state.State) {
```

Comment:
I just realized this: we should group all the handlers under `setupDebugHandlers`. In that function, simply add your `http.HandleFunc...`

Reply:
Should I do it in this PR or make a new one?

Reply:
Same one yeah